storagenode/pieces/store.go

```diff
@@ -132,20 +136,35 @@ type Store struct {
 	reservedSpace int64
 }
 
+// SpaceUsedLive stores the live totals of used space for all
+// piece content (not including headers) and also total used space
+// by satellites
+type SpaceUsedLive struct {
```

Comment:
Let's implement this caching as wrapper instead of complicating the existing code.

Reply:
a wrapper around what?

Reply:
added commit to address this comment and add a wrapper: https://github.com/storj/storj/pull/2707/commits/296ccc47ff0e8cf56f49d541cfe6d56ea595bbda